views/layouts/main.blade.php

```diff
@@ -3,6 +3,11 @@
 
 <head>
     @include('twill::partials.head')
+		<style>
+			.fieldset__content > hr {
```

Comment:
Same here. By the way if you're trying to remove the separator from the vlock_edktor field there is an option for that: `withoutSeparator`.